ParallelPrimitives/RadixSortConfigs.h

```diff
@@ -38,8 +20,26 @@ constexpr auto SINGLE_SORT_WG_SIZE{ 128 };
 
 static_assert( BIN_SIZE % 2 == 0 );
 
-// Notice that, on some GPUs, the max size of a GPU block cannot be greater than 256
-static_assert( DEFAULT_COUNT_BLOCK_SIZE % DEFAULT_WARP_SIZE == 0 );
-static_assert( DEFAULT_SCAN_BLOCK_SIZE % DEFAULT_WARP_SIZE == 0 );
+constexpr int WARP_SIZE = 32;
+
+constexpr int RADIX_SORT_BLOCK_SIZE = 4096;
```

Comment:
Does this mean that in this version, the block size and the parameters below are all fixed again ?

Reply:
Yes. but even with fixed size, the temporal allocation is fixed.